overview.tex

```diff
@@ -1,34 +1,35 @@
 \subsection*{Introduction}
 
-    When creating a software, it is usual to evaluate it regarding a set of
-    information of interest
-    --- such as CPU time, number of
+    When creating a software, it is usual measure a set of
+    information of interest regarding performance
+    --- for instance: CPU time, number of
     functions evaluations, number of iterations, memory usage, accuracy, or
-    others --- in a
-    process called \emph{benchmarking}.
+    others.  \emph{Benchmarking} is the process of measuring those performance
+    informations of a software relatively to similar ones.
```

Comment:
->"measuring **the** performance **information** of a software **relative** to similar ones.